packages/components/app/styles/components/side-nav/main.scss

```diff
@@ -160,6 +205,12 @@
   padding-right: var(--token-side-nav-wrapper-padding-horizontal);
   padding-bottom: 8px; // by design
   padding-left: var(--token-side-nav-wrapper-padding-horizontal);
+
+  .hds-side-nav--is-minimized & {
+    padding-top: var(--token-side-nav-wrapper-padding-vertical-minimized);
```

Comment:
should also these paddding values have a smooth transition? 🤔

Reply:
Added the `SideNav` default transition values to both the logo size and the wrapper padding in c7f5bca5fadd6cd9e27bdfc44163d2357894f8b8